backend/tests/apps/nest/models/user_test.py

```
@@ -0,0 +1,36 @@
+
+
+class TestUserModel:
+    """Unit tests for User model without DB."""
+
+    def test_model_fields(self):
+        assert isinstance(User._meta.get_field("github_id"), models.CharField)
+        assert User._meta.get_field("github_id").max_length == 255
+        assert User._meta.get_field("github_id").unique is True
+
+        assert isinstance(User._meta.get_field("username"), models.CharField)
+        assert User._meta.get_field("username").max_length == 150
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `150` with a constant variable

(PLR2004)